msat-set-service-repo-url-metadata.sh

```diff
@@ -0,0 +1,58 @@
+#! /bin/bash -e
+
+while [ -n "$1" ]
+do
+	case $1 in
+	--api-endpoint)
+		api_endpoint=${2?}
+		shift
+		;;
+	*)
+		break
+		;;
+	esac
+	shift
+done
+
+if [ -z $api_endpoint ]
```

Comment:
Eliminate this conditional logic by moving `api_endpoint="https://platform.microservices.io/graphql"` to before the while